CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/templates/PluginTags.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.templates;
+
+/**
+ *
+ * @author aldebaran30701
+ */
+public class PluginTags {
+    
+    public static final String ANALYTIC = "ANALYTIC";
+    public static final String LOW_LEVEL = "LOW LEVEL";
+    public static final String MODIFY = "MODIFY";
+    public static final String GENERAL = "GENERAL";
+    public static final String SELECT = "SELECT";
+    public static final String SEARCH = "SEARCH";
+    public static final String CREATE = "CREATE";
+    public static final String DEVELOPER = "DEVELOPER";
+    public static final String EXPERIMENTAL = "EXPERIMENTAL";
+    public static final String WELCOME = "WELCOME";
+    public static final String DELETE = "DELETE";
+    public static final String IMPORT = "IMPORT";
+    public static final String EXPORT = "EXPORT";
+    public static final String UTILITY = "UTILITY";
+    public static final String VIEW = "VIEW";
+    
+    
+    private PluginTags(){
+        throw new IllegalStateException("Utility class");
```

Comment:
I would consider changing the text to "Constants class" or similar since by our definition it's not technically a Utility class

Reply:
Done